packages/api/src/external/carequality/api.ts

```diff
@@ -0,0 +1,14 @@
+import { Config } from "../../shared/config";
+import { Carequality, APIMode } from "@metriport/carequality-sdk";
+
+/**
+ * Creates a new instance of the Carequality API client.
+ * @param apiKey Optional, API key to use for authentication. If not used, the API key will be retrieved from the environment variables.
+ * @param dev Optiona, creates the API in dev mode.
+ * @returns Carequality API.
+ */
+export function makeCarequalityAPI(apiKey?: string, dev = false): Carequality {
+  const cqApiKey = apiKey ?? Config.getCQApiKey();
+  if (dev) return new Carequality(cqApiKey, APIMode.dev);
```

Comment:
Don't we want to use `Config.isCloudEnv()` or similar instead?

Reply:
Yeah, that's probably better

Reply:
actually, the other routes on `internal-cq` are not going thru this logic, so ill just get rid of it